tests/extensions/inline/bold/full/invalid/whitespace-only.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="../basic.rng" schematypens="http://relaxng.org/ns/structure/1.0"?>
+<?xml-model href="../basic.rng" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<?expected-error line="6" message="Bolds must not be empty" node="/libero:bold"?>
+
+<libero:bold xmlns:libero="http://libero.pub">
+    
```

Comment:
It'd be good to keep actual spaces here (not just line breaks). You'll probably need to do it outside of your IDE, [I've been there](https://github.com/libero/php-coding-standard/pull/32#discussion_r235732768)!

Reply:
I checked and I have 4 spaces of whitespace on line 7. The same as in `tests/extensions/inline/italic/full/invalid/whitespace-only.xml`

Reply:
There is indeed; the diff view doesn't show it for some reason!